app/views/admin/users/index.html.slim

```diff
@@ -0,0 +1,23 @@
+section.section
+  .columns
+    .column
+      h1.title Slack Users
+
+      table.table.is-hoverable
+        thead
+          tr
+            th Admin
+            th First Name
+            th Last Name
+            th Created At
+            th Email
+            th Timezone
+        tbody
+          - @users.each do |user|
+            tr[class = (user.is_deleted? ? "has-text-grey-light" : "")]
+              td= (user.is_admin ? "&#x2713;".html_safe : "")
```

Comment:
what's this entity here? can it not just be a unicode character literal?

Reply:
It's a checkmark.  I just pulled an HTML code for it.  I can just copy in ✓ but I'm not very well versed in front-end so I wasn't sure what support would be like, because the last time I wrote front-end code on a regular basis I had to use hex codes, 400 years ago on Geocities. ¯\_(ツ)_/¯ I'll fix it.

Reply:
the template should have the meta encoding set to utf-8 to make that work (for non-ancient browsers/oses)

Reply:
if it isn't set, i consider that a bug that can be fixed